packages/core/services/FileService/DatabaseFileService/test/DatabaseFileService.test.ts

```diff
@@ -154,5 +154,45 @@ describe("DatabaseFileService", () => {
             // Assert
             expect(modifiedSQL).to.include("OFFSET 0 LIMIT 3");
         });
+
+        it("correctly applies AND vs OR clauses", () => {
+            const selectionsWithOR = [
+                {
+                    indexRanges: [{ start: 0, end: 2 }], // File range
+                    filters: {
+                        Structure: ["structure1", "structure2"],
+                    },
+                },
+            ];
+            const selectionsWithAND = [
+                {
+                    indexRanges: [{ start: 0, end: 2 }], // File range
+                    filters: {
+                        "Cell Line": ["AICS-01"],
+                        Structure: ["structure1"],
+                    },
+                },
+            ];
+            // Make a separate SQLBuilder for comparison
+            const sqlBuilderAND = new SQLBuilder().select("*").from("mock_source");
+
+            // Act
+            DatabaseFileService.applySelectionFilters(sqlBuilder, selectionsWithOR, [
+                "mock_source",
+            ]);
+            const modifiedSQLWithOR = normalizeSQL(sqlBuilder.toSQL());
+            DatabaseFileService.applySelectionFilters(sqlBuilderAND, selectionsWithAND, [
+                "mock_source",
+            ]);
+            const modifiedSQLWithAND = normalizeSQL(sqlBuilderAND.toSQL());
+
+            // Assert
+            // Uses OR within single filter type
+            expect(modifiedSQLWithOR).to.include("OR");
```

Comment:
Would this test detect the "()" issue the change is fixing?  Would expect(modifiedSQLWithOR).to.include(" OR ")
allow detection of the old code?  If we can match more of the generated SQL for correctness, that might make both the test and the intent of the code more easily readable to future workers.

Reply:
This particular test was intended to catch the other bug, where OR was being applied where AND should've been applied. I'll see if I can add a test for the other change as well

Reply:
Remembering now that the other unit test I fixed actually does address the `) OR (` change. The problem was that while we did have a unit test that matched more of the generated SQL code, it was matching the wrong thing (our logic and expectation was wrong, not the code)